test/browser.test.js

```diff
@@ -16,6 +21,15 @@ afterEach(() => {
     browserslist.data = originData;
 });
 
+it('selects all versions of browser', () => {
+    expect(browserslist('all ie')).toEqual(['ie 11', 'ie 10', 'ie 9']);
+});
+
+it('selects all versions of all browsers', () => {
```

Comment:
I think breaking line after 'edge 13' doesn't correspond to other code style. What about this version:
```
it('selects all versions of all browsers', () => {
    expect(browserslist('all')).toEqual([
        'edge 15', 'edge 14', 'edge 13', 'edge 12', 'ie 11', 'ie 10', 'ie 9'
    ]);
});
```